components/detail/tabs/Tabs.tsx

```diff
@@ -2,20 +2,19 @@ import React, { useRef, useState } from 'react';
 import { TabsHeader, Tab, TabIndicator, TabsWrapper } from './Tabs.style';
 
 function Tabs() {
-  const [currentTab, setCurrentTab] = useState('');
   const indicatorRef = useRef<any>(null);
+  const tabsHeaderRef = useRef<any>(null);
```

Comment:
```suggestion
  const tabsHeaderRef = useRef<HTMLDivElement>(null);
```

처럼 타입 지정해주어도 좋을 것 같아요!

https://driip.me/7126d5d5-1937-44a8-98ed-f9065a7c35b5